src/utils/interface.ts

```diff
@@ -0,0 +1,27 @@
+interface MovieCard {
+  movieImageUrl: string;
+  movieTitle: string;
+  movieDescription: string;
```

Comment:
Nit: Can simplify the property name.
```suggestion
  posterUrl: string;
  title: string;
  description: string;
```

Reply:
I have updated `MovieCard` interface (https://github.com/krushna-simform/filmfusion/pull/2/commits/d2761b36c892f6191af7c528371f29c33aafc64b)